src/Preflight/RedispatchToSiteLocal.php

```diff
@@ -58,7 +58,12 @@ function ($item) {
         );
         $command .= ' ' . implode(' ', $args);
         passthru($command, $status);
-        return $status;
+
+        // The $status variable is the exit status of the process, which is an
+        // integer. This method is supposed to return true when successful, so
+        // we need to convert it to a boolean accordingly.
+
+        return $status ? false : true;
```

Comment:
This function should always return true if it gets to this point. The comment in the header that says "and was returned successfully" was in error. See my other comment for a description of how the code used to work.

I think that the best fix would be to change this function to use `return [false, 0];` wherever it currently uses `return false`, and change this line to `return [true, $status];`. Then, `doRun` must record both return values, call them `$preflightDidRedispatch` and `$status` or something similar. It should then return `$status` if `$preflightDidRedispatch` is true.

Reply:
Thanks for the feedback, this should be fixed now.